homeassistant/components/iaqualink/__init__.py

```diff
@@ -0,0 +1,107 @@
+"""Component to embed Aqualink devices."""
+import asyncio
+import logging
+
+from aiohttp import CookieJar
+import voluptuous as vol
+
+
+from iaqualink import AqualinkClient, AqualinkLoginException, AqualinkThermostat
+
+from homeassistant import config_entries
+from homeassistant.components.climate import DOMAIN as CLIMATE_DOMAIN
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.helpers.aiohttp_client import async_create_clientsession
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.typing import ConfigType, HomeAssistantType
+
+from .const import DOMAIN
+
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTR_CONFIG = "config"
+PARALLEL_UPDATES = 0
+
+CONFIG_SCHEMA = vol.Schema(
+    {
+        DOMAIN: vol.Schema(
+            {
+                vol.Required(CONF_USERNAME): cv.string,
+                vol.Required(CONF_PASSWORD): cv.string,
+            }
+        )
+    },
+    extra=vol.ALLOW_EXTRA,
+)
+
+
+async def async_setup(hass: HomeAssistantType, config: ConfigType) -> None:
+    """Set up the Aqualink component."""
+    conf = config.get(DOMAIN)
+
+    hass.data[DOMAIN] = {}
+    hass.data[DOMAIN][ATTR_CONFIG] = conf
```

Comment:
We don't seem to use the config stored here?

Reply:
Probably a pattern copied from another component.